bundles/ImageMagick

```diff
@@ -1,8 +1,8 @@
 # [TITLE]: ImageMagick
-# [DESCRIPTION]: ImageMagick
+# [DESCRIPTION]: Create, edit, or compose bitmap images. 
```

Comment:
I noticed that you sometimes have spaces at the end of lines, you don't need to fix this set of PRs but keep an eye out for that.

Reply:
OK.  Will do so.